vspace/src/main/webapp/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Class-Path: 
+

```

Comment:
what is this for?

Reply:
Somehow by mistake this file came in. I deleted this now.